policyengine_uk/parameters/gov/dwp/extended_childcare_entitlement/eligibility/second_entitlement.yaml

```diff
@@ -0,0 +1,29 @@
+description: The Department for Education provides the extended childcare entitlement to children this age or older
```

Comment:
I'd just have two parameters, min and max age. Your description is inconsistent anyway with this current format (this age or older-> bracket?).